doc/source/user_guide/numpyro_refitting_xr_lik.ipynb

```diff
@@ -141,7 +141,8 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "We have defined a dictionary `sample_kwargs` that will be passed to the `SamplingWrapper` in order to make sure that all refits use the same sampler parameters. We follow the same pattern with `az.from_numpyro`."
+    "We have defined a dictionary `sample_kwargs` that will be passed to the `SamplingWrapper` in order to make sure that all\n",
+    "refits use the same sampler parameters. We follow the same pattern with {func}`~az.from_numpyro`."
```

Comment:
Please change {func}`~az.from_numpyro` to {func}`az.from_numpyro <arviz.from_numpyro>`

Reply:
using `` {func}`~arviz.from_numpyro` `` or `` {func}`arviz.from_numpyro` `` is perfectly good enough, no need to use custom text if it's only for the `az.` part. It is important to change what is used now though because sphinx doesn't use alias when  importing, so it won't generate a clickable link if using `az.from_numpyro` as reference.

Reply:
So um should I change anything or is it good to go?

Reply:
it is already good to go, just wanted to comment to ease future work

Reply:
oh okay sure, can you tell me when it'll be merged though